### PR TITLE
Balancer Forks: Fix bugs/exception with stableFactoryPools

### PR DESCRIPTION
**Pull request by harsh9200**

### CODE REVIEW

File: subgraphs/balancer-forks/src/modules/Rewards.ts [hunk 1]
```diff
@@ -116,7 +114,13 @@ export function updateFactoryRewards(
   );
   pool.save();
   //////////////////////////////////////////////////////////////////
+}
 
+export function updateFactoryRewards(
+  poolAddress: Address,
+  gaugeAddress: Address,
+  block: ethereum.Block
+): void {
```
Comment (steegecs) [inline subgraphs/balancer-forks/src/modules/Rewards.ts:123]: Won't this miss the stakedOutTokenAmount of the side chains though? Are you thinking there is not a way to calculate this for chains that are not Ethereum mainnet? My impression was that the updateControllerRewards() will only end up updating the stakedOutputTokenAmount for Ethereum mainnet if it is left there, because this code will only get executed on Ethereum mainnet.

File: subgraphs/balancer-forks/src/prices/common/constants.ts [hunk 2]
```diff
@@ -46,6 +46,22 @@ export const WHITELIST_TOKENS_LIST: string[] = [
   "CRV",
 ];
 
+export const USD_PEGGED_TOKENS: string[] = [
```
Comment (steegecs) [inline subgraphs/balancer-forks/src/prices/common/constants.ts:49]: Why did you add these here?

Comment (steegecs) [reply subgraphs/balancer-forks/src/prices/common/constants.ts:49]: *All of the USD_PEGGED_TOKENS.

Comment (harsh9200) [reply subgraphs/balancer-forks/src/prices/common/constants.ts:49]: These are all the stable tokens on which the Price lib failed to fetch a correct price.

**Review by steegecs — COMMENTED** [review]
Okay - Everything looks good to me!

**Review by steegecs — APPROVED** [review]
